### ISSUE TITLE
TV_OFF not working with PiSignage Image V5.0.5 / Wayland

### ISSUE THREAD

**Issue by pillgrim** [colloqi/piSignage#241]
I have noticed that the monitor does not go into standby mode despite the tv_off playlist

Can you please check this

Many thanks :-)

**Comment by AbhijithBn** [comment]
Hi

TV_OFF works only if HDMI-cec is supported/enabled on your Display.

Can you please reach out to us at support at pisignage.com with these details?
1) Username on pisignage.com
2) Player ID having this issue

**Comment by pillgrim** [comment]
Hi @AbhijithBn 

Thanks for your reply.

Our players connecting to the pisignage open source server. 
I use for many years this setup with the player2 image 4.x.x and the tv or monitor goes to standby, if the playlist is tv_off is active. with the player2 image 5.x.x the standby mode doesnt work anymore.

thanks for checking

**Comment by wrldtvlr** [comment]
FWIW I don't think TV implementation of CEC is very reliable. I just bought a Bose Sound bar which has about 6 or 7 different CEC options. Had to try all of them to find one that worked with my Samsung TV.

**Comment by pillgrim** [comment]
Thanks @wrldtvlr for your reply.

maybe I'm wrong about cec. The monitor stays on all the time with release 5.x.x. but with release 4.8.x the monitor no longer receives an hdmi signal if the tv_off playlist goes on

**Comment by wrldtvlr** [comment]
I agree, it's frustrating. We have three 2022 LG TVs working with PiSignage. We have to leave them on 24/7 because CEC off/on was unreliable.

**Comment by pillgrim** [comment]
do you use vcgencmd or tvservice for hdmi controlling? this seems not working anymore with the bookworm release

**Comment by weswin** [comment]
Just for Info,

 

We had a similar issue, but our TV (older Panasonic) had a timer built in so we used that, if that didn’t work we we’re going to put the TV on a lighting  timer.

**Comment by pillgrim** [comment]
I have tried the following commands under the Bookworm version to turn the screen on and off:
wlr-randr --output HDMI-A-1 --on
wlr-randr --output HDMI-A-1 --off
This allows the monitor to go into standby mode as no signal is coming in. perhaps this helps
